NuKeeper.AzureDevOps/AzureDevOpsSettingsReader.cs

```diff
@@ -13,44 +13,35 @@ public class AzureDevOpsSettingsReader : BaseSettingsReader
         private const string UrlPattern = "https://dev.azure.com/{org}/{project}/_git/{repo}/";
 
         private readonly IGitDiscoveryDriver _gitDriver;
-        private bool _isLocalGitRepo;
 
-        public AzureDevOpsSettingsReader(IGitDiscoveryDriver gitDriver, IEnvironmentVariablesProvider environmentVariablesProvider)
+        public AzureDevOpsSettingsReader(IGitDiscoveryDriver gitDriver,
+            IEnvironmentVariablesProvider environmentVariablesProvider)
             : base(environmentVariablesProvider)
         {
             _gitDriver = gitDriver;
         }
 
         public override bool CanRead(Uri repositoryUri)
         {
-            if (repositoryUri == null)
-            {
-                return false;
-            }
+            if (repositoryUri == null) return false;
```

Comment:
This goes against the style used in NuKeeper. We might need a style file to enforce this.

Reply:
Oh yeah, also noticed it. Accidentally clicked `Code cleanup` when using Rider. Will fix.

Reply:
BTW - it's fixed.